drivers/hashmap/hashmap.go

```diff
@@ -110,7 +110,7 @@ func (db *Database) Setup() error {
 	defer db.Unlock()
 
 	// check file and create if it does not exist
-	file, err := os.OpenFile(db.config.Filename, os.O_RDONLY|os.O_CREATE, 0755)
+	file, err := os.OpenFile(db.config.Filename, os.O_RDONLY|os.O_CREATE, 0644)
```

Comment:
Good catch, I think we should go with 640 though as this probably should also not be world readable.

Reply:
Cool, just pushed that change. I also updated the permission on the test files in `testdata`, but git didn't show them as modified